dll/win32/ws2_32/src/startup.c

```diff
@@ -105,6 +151,86 @@ WSACleanup(VOID)
     return ErrorCode;
 }
 
+/* WSACallThread will call a WSA function in an other thread and returns the result  */
+
+INT
+WINAPI
+WSACallThread(IN int funcId, 
+               IN LPVOID arg1, 
+               IN LPVOID arg2)
```

Comment:
Unnecessary blank space on function parameters.

Reply:
You are sharp, didn't notice that before!